opentelemetry-user-events-metrics/src/exporter/mod.rs

```diff
@@ -67,16 +76,16 @@ impl Debug for MetricsExporter {
 #[async_trait]
 impl PushMetricsExporter for MetricsExporter {
     async fn export(&self, metrics: &mut ResourceMetrics) -> Result<()> {
-        if tracepoint::enabled() {
+        if self.trace_point.enabled() {
             let proto_message = transform_resource_metrics(metrics);
 
             let mut byte_array = Vec::new();
             let _encode_result = proto_message
                 .encode(&mut byte_array)
                 .map_err(|err| MetricsError::Other(err.to_string()))?;
-            let result = tracepoint::write(byte_array.as_slice());
+            let result = tracepoint::write(&self.trace_point, byte_array.as_slice());
```

Comment:
As mentioned in the comments for write, the result of write is for debugging. It can fail for various reasons, mainly if there is nobody listening for the event. These failures should not affect the running program and are only interesting for diagnostics.

Reply:
Removed.